examples/ABTests.py

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
Example is actually pretty good. I'm going to write an actual bot interface once the library reaches 80% API completion across all roots. For now, just make sure that the endpoints actually work & can be used with or without a bot interface (compatibility w/ Discord.py, etc).